py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Optional, Tuple, Union
+
+import tensorrt as trt
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion.converter_utils import get_axes_for_reduce_op
+from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+
+def amax(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dim: Union[int, Tuple[int]],
+    keep_dims: Optional[bool] = False,
```

Comment:
The `Optional` here can be removed, since `keepdim` is a mandatory argument, as per [this documentation](https://github.com/pytorch/pytorch/blob/a408920817ac3852d27fc96ba85109ff4c981d52/aten/src/ATen/native/native_functions.yaml#L3643-L3646)